conn.go

```diff
@@ -318,8 +343,6 @@ func (c *Conn) handleAuth(arg string) {
 
 	if c.User() != nil {
 		c.WriteResponse(235, "Authentication succeeded")
-
-		c.msg = &message{}
```

Comment:
The sasl handlers are expected to call `SetUser`, which now sets the message too.